scripts/lib.star

```diff
@@ -441,6 +443,33 @@ def build_frontend_step(edition, is_downstream=False):
         ],
     }
 
+def generate_package_docs(edition, lint=False):
```

Comment:
Can we just merge the command from this step into the build-docs-website step instead? I don't see any benefit to complicating the build graph with another step.

Reply:
Yes, I will change that 👍

Reply:
Never mind, leave it be for now since the build image is (probably) incompatible in build-docs-website.